OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/pdf/PDFDisplay.java

```diff
@@ -36,9 +36,13 @@ public JPanel drawPreviewOnJPanel() {
         Image generatedImage = PDFPreviewBuilder.buildPDFPreview(selectedPage, viewZoom);
         
         if (generatedImage != null) {
-            ImageIcon icon = new ImageIcon(generatedImage);
-            JLabel picLabel = new JLabel(icon);
-            pdfImagePanel.add(picLabel);
+            try {
+                ImageIcon icon = new ImageIcon(generatedImage);
+                JLabel picLabel = new JLabel(icon);
+                pdfImagePanel.add(picLabel);
+            } catch (OutOfMemoryError oome) {
```

Comment:
Catching OutOfMemoryError is pointless. All it really does is preventing the stacktrace from being shown to the user but we cannot predict the system's behaviour.